src/api/main.py

```diff
@@ -209,6 +208,21 @@ def log_request(response):
         data["status_code"] = response.status_code
         if data.get("username"):
             log_manager = LogManager()
+            if data.get("args", {}).get("domain_id"):
```

Comment:
Perhaps a consolidated `args = data.get("args", {})` or similar would be appropriate since you're doing that repeatedly. You could then replace the later `data["args]` calls with this new variable as well to cut down on repetition.

Reply:
agreed. resolved with [3a6799a](https://github.com/cisagov/domain-manager-api/pull/515/commits/3a6799afe4b43911002c74e68305625eac9f56ad)